equivalences.csv

```diff
@@ -1672,3 +1673,4 @@ SIGNOR,SIGNOR-PF6,AMPK_alpha
 SIGNOR,SIGNOR-PF7,p14_3_3
 SIGNOR,SIGNOR-PF8,FGF
 SIGNOR,SIGNOR-PF9,BMP_receptor
+mirbase.family,MIPF0000002,MIRLET7

```

Comment:
is this the way you want to stylize it?

Reply:
It's probably arbitrary but we have had a convention to capitalize these name spaces so maybe we could do that. Having said that I'm tempted to, at some point, standardize equivalences here to use proper curie prefixes, which I am sure you would be pleased about :)

Reply:
Addressed with https://github.com/sorgerlab/famplex/pull/134/commits/b43db8904cb8246fb8db3e907ccdb2f6dc72c220 and new export with https://github.com/sorgerlab/famplex/pull/134/commits/0ed9f4a6770bd0104443ed72396051e0f1700041. Sorry for the delay

Reply:
Thanks!